distributed/recreate_exceptions.py

```diff
@@ -91,7 +91,7 @@ def _get_futures_error(self, future):
             raise gen.Return((function, args, kwargs, deps))
         else:
             function, args, kwargs = _deserialize(task=task)
-            raise gen.Return((function, args, kwargs, []))
+            raise gen.Return((function, args, kwargs, deps))
```

Comment:
@martindurant any objection to this change?

Reply:
no objection

Reply:
Thanks!